sympy/polys/matrices/domainmatrix.py

```diff
@@ -328,7 +319,7 @@ def from_list_sympy(cls, nrows, ncols, rows, **kwargs):
 
         domain, items_domain = cls.get_domain(items_sympy, **kwargs)
 
-        domain_rows = [[items_domain[ncols*r + c] for c in range(ncols)] for r in range(nrows)]
+        domain_rows = [[items_domain[ncols * r + c] for c in range(ncols)] for r in range(nrows)]
```

Comment:
Changes like this are not always wanted so if your editor is doing this automatically then it would be better to disable that. I don't think that always putting spaces around an operator improves the code. Generally PEP 8 is (loosely) followed which says:

> If operators with different priorities are used, consider adding whitespace around the operators with the lowest priority(ies)

Here `+` has a lower priority than `*` so `ncols*r + c` helps to emphasise the order of operations.

Tools like black have been rejected for use in SymPy precisely because they are not good for mathematical expressions e.g.
```python
a*x**3 + b*x**2 + c*x + d
```
gets turned into
```python
a * x ** 3 + b * x ** 2 + c * x + d
```
which ruins all sense of precedence.

Also apart from very localised changes if there is a desire to reformat then it should be a separate PR.

Reply:
Yeah, I think my formatter does that, I should have checked, I will try to undo it. I need to install something that will let me view diffs before I push, so I will see this sort of thing.

Reply:
Can you not just run `git diff`? How are you using `git`?

Reply:
I am using git in the terminal and running diff is not really part of my workflow because I find it really hard to look at, it just looks crazy to my eyes, I don't know how else to say that. I have to spend a lot of time weeding through the output to really see what I did. I used to have a Mac and used sourcetree, which was quite nice, but I am on Ubuntu now and they don't make a version for Linux. I am going to try GitKraken because my searching is telling me that seems to be the closest to sourcetree for linux.